### ISSUE TITLE
Weird formatting behavior when line wraps in an assignment to a mapping with storage keyword

### ISSUE THREAD

**Issue by petejkim** [prettier-solidity/prettier-plugin-solidity#282]
Expected:
```solidity
contract Test {
    mapping(address => mapping(address => uint256)) private _foo;

    function foo() external {
        for (uint256 i = 0; i < 5; i++) {
            address reallyReallyLongName = address(0);
            mapping(address => uint256) storage foo
              = _foo[reallyReallyLongName];
            foo[address(0)] = 1234;
        }
    }
}
```

Got:
```solidity
contract Test {
    mapping(address => mapping(address => uint256)) private _foo;

    function foo() external {
        for (uint256 i = 0; i < 5; i++) {
            address reallyReallyLongName = address(0);

                mapping(address => uint256) storage foo
             = _foo[reallyReallyLongName];
            foo[address(0)] = 1234;
        }
    }
}
```

**Comment by petejkim** [comment]
It appears that it's not limited to mappings

**Comment by Janther** [comment]
Thank you,

it's a known bug related to initial values in a variable declaration. I haven't had time to go into it but it's on my TODOs.

**Comment by fvictorio** [comment]
Fixed in #379.
